From 0efa7abffe5fffbde8c457d3c8fafbdde0bb6e4f Mon Sep 17 00:00:00 2001
From: Enrico Jorns <ejo@pengutronix.de>
Date: Sun, 21 Aug 2022 19:21:51 +0200
Subject: [PATCH 1/1] caencoder: remove sys/mount.h to fix build against glibc
 2.36

Since glibc 2.36, the mount.h is no longer compatible with the linux
mount.h and inclusion of both will cause definition conflicts:

https://sourceware.org/glibc/wiki/Release/2.36#Usage_of_.3Clinux.2Fmount.h.3E_and_.3Csys.2Fmount.h.3E

Fix this by removing inclusion of sys/mount.h since inclusion of
linux/fs.h already pulls in linux/mount.h.

Signed-off-by: Enrico Jorns <ejo@pengutronix.de>
---
 src/caencoder.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/caencoder.c b/src/caencoder.c
index 5999104..0d53575 100644
--- a/src/caencoder.c
+++ b/src/caencoder.c
@@ -12,7 +12,6 @@
 #include <string.h>
 #include <sys/acl.h>
 #include <sys/ioctl.h>
-#include <sys/mount.h>
 #include <sys/stat.h>
 #include <sys/sysmacros.h>
 #include <sys/xattr.h>
-- 
2.37.2

